rules/cross-platform/execution_python_script_in_cmdline.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2021/01/13"
+maturity = "production"
+updated_date = "2021/01/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when a Python script is executed using command line input. Attackers often use this method to execute
+malicious scripts and avoiding writing it to disk.
+"""
+false_positives = ["Legitimate Python scripting activity."]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Python Script Execution via Command Line"
+risk_score = 47
+rule_id = "ee9f08dc-cf80-4124-94ae-08c405f059ae"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "macOS", "Windows", "Threat Detection", "Execution"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+ process.name:(Python* or python*) and process.args:("-c" and import*sys,*)
```

Comment:
I like this approach as a rule. This one might just be too brittle and overfitted and better off waiting for eql regex. Some common examples that break this:

```
python -c "# leading comment; import    sys;  print(sys.version_info)"
python -c "from sys import version_info; print(version_info)"
python -c "cmd = 'import sys'; exec(cmd); print(sys.version_info)"

```

Reply:
I know, as-is it matches on default metasploit python and other similar tools one-liner payload, we can add regex as a tuning for this and similar rules but I think it won't harm to have there as a placeholder and covering default stuff in the mean-time.

Reply:
It definitely can provide value in specific use cases. I would say to update the name and description to more accurately target those use cases and then we can eventually write a more generic _Python Script Execution via Command Line_ rule once we have regex. May also be worth a note in the description that it is not a catch all and will miss anything other than the specific formatted python

Reply:
@brokensound77 converted to eql 
```
process where event.type in ("start", "process_started") and
 process.name : "python*" and process.args : "-c" and process.args : "*import*sys*"
```
we have lot of rules also in eql-rules repo that have similar resilience level, and I can give you lot examples if needed :)

Reply:
I know that we have many similar style rules, just helps to keep balance to not inundate users by having to heavily tune too many rules.